launcher/src/main/java/org/wso2/carbon/launcher/Constants.java

```diff
@@ -28,7 +28,8 @@
     public static final String CARBON_START_TIME = "carbon.start.time";
 
     public static final String PROFILE = "profile";
-    public static final String DEFAULT_PROFILE = "profiles/default";
+    public static final String PROFILES = "profiles";
```

Comment:
But this is for coding purposes in my opinion. This constant is used to identify the profiles folder. We don't use this constant for the creation of profiles.